### PR TITLE
add brynet/1.11.0

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **lib/1.0**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/brynet/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class BrynetConan(ConanFile):
+    name = "brynet"
+    description = "Header Only Cross platform high performance TCP network library using C++ 11."
+    license = "MIT"
```
Comment (prince-chrismc) [inline recipes/brynet/all/conanfile.py:10]: It also says "996 limited prohibited" I'm the readme

Comment (SpaceIm) [reply recipes/brynet/all/conanfile.py:10]: I don't understand, I see something related to chinese law about working hours, but it's not a license.

Comment (prince-chrismc) [reply recipes/brynet/all/conanfile.py:10]: Screenshot_20210605-143856_Chrome

https://github.com/996icu/996.ICU/blob/master/LICENSE

Comment (SpaceIm) [reply recipes/brynet/all/conanfile.py:10]: yes I see, but it's the license of brynet

Comment (prince-chrismc) [reply recipes/brynet/all/conanfile.py:10]: Could have fooled me.

**Review by prince-chrismc — APPROVED** [review]

**Review by madebr — APPROVED** [review]

File: recipes/brynet/all/conanfile.py (hunk 1 above)
Comment (madebr) [reply recipes/brynet/all/conanfile.py:10]: It's weird nonetheless. Maybe it's worth to create an issue for them to comment on this?

**Review by jgsogo — APPROVED** [review]

**Comment by eeyrw** [comment]
The brynet have updated to 1.11.1. How to make it synchronize to conan repository?

**Comment by ericLemanissier** [comment]
You can go to https://qchateau.github.io/conan-center-bot/#/updatable to create the Pull Reques for the update

**Pull request merged**